### ISSUE TITLE
Service navigation web component `default-language` can't be changed

### ISSUE THREAD

**Issue by ice-blaze** [oblique-bit/oblique#132]
### Oblique version

11.3.2

### Is this a regression?

Yes

### Description

```true
Depending on the different ways to pass a value, the `default-language` isn't taken into account. Also if you change the language it doesn't seem to be udpate.
```

### Reproduction

```html
<ob-service-navigation-web-component
    default-language="it"        
/>
```

Solutions that doesn't work
```html
<ob-service-navigation-web-component
    [defaultLanguage]="'it'"        
/>
```

```html
<ob-service-navigation-web-component
    default-language="{{'it'}}"        
/>
```

### Expected behaviour

```true
Be able to use the the different angular ways to pass a value through the input.
When the value change, the component language should also change.
```

**Comment by gillerr** [comment]
Hi @ice-blaze 
`default-language`, `language-list` and `environment` and only evaluate on component initialization because they shouldn't change during the component's life cycle.

`environment` may not change because everything else depends on this.
`language-list` are the languages supported by the application, I can't think of a use case where it would be necessary to change them at run time.
`default-language` is the fallback language when a translation is missing, either from the current language. Again, I can't think of a use case to change than at run time.

By the way Oblique itself don't allow languages to be changed at run time either.

**Comment by ice-blaze** [comment]
Then there is still the question of using variables that is still open. I'll try it in a clean project to be sure it wasn't on my side an angular issue.

**Comment by gillerr** [comment]
this has been fixed with Oblique 12.0.0
